src/test/java/core/basesyntax/daotest/ReadFromFileTest.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.daotest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.ReadFromFile;
+import core.basesyntax.dao.daoimpl.ReadFromFileImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ReadFromFileTest {
+    public static final int EXPECTED_SIZE_OF_EMPTY_LIST = 0;
+    public static final String FILE_PATH = "src/test/resources/inputOk.csv";
+    public static final String EMPTY_FILE_PATH = "src/test/resources/inputEmpty.csv";
+    public static final String WRONG_FILE_PATH = "src/test/resources/blablabla";
+    public static final String NULL_FILE_PATH = null;
+    private ReadFromFile reader;
+
+    @Test
+    void readFromFile_Ok() {
+        List<String> expected = new ArrayList<>();
+        fillTransactionsList(expected);

```

Comment:
You basically need to compare against a constant, so instead of using those two lines in your tests, you can compare content against a field (created using `List.of` for example)